mslice/util/mantid.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import (absolute_import, division, print_function)
+
+import mantid.simpleapi as s_api
+from mantid.api import AlgorithmFactory
+from mslice.models.workspacemanager.workspace_provider import add_workspace
+from mslice.models.projection.powder.make_projection import MakeProjection
+from mslice.workspace import wrap_workspace
+from mslice.workspace.base import WorkspaceBase as Workspace
+
+
+def initialize_mantid():
+    AlgorithmFactory.subscribe(MakeProjection)
+    s_api._create_algorithm_function('MakeProjection', 1, MakeProjection())
+
+
+def run_alg(alg_name, output_name=None, store=True, **kwargs):
```

Comment:
I would say it might be confusing in the code to have a both a `run_alg` and `run_algorithm` function - it's not immediately obvious what the difference is.

How about a single `run_algorithm` that can accept either a string or algorithm function?